config/fund_loader_config/cof/cof_r4.py

```diff
@@ -293,3 +519,66 @@
         "eoi_decision_schema": None,
     }
 ]
+
+round_config_w2 = [
```

Comment:
Looks good but you may need to update with the new attributes I'm adding here: https://github.com/communitiesuk/funding-service-design-fund-store/pull/295

Reply:
Nice spot, fix added, thanks Thomas.